source/reference/versioning.txt

```diff
@@ -27,12 +27,14 @@ version number for {+c2c-product-name+}:
 - Patch number: This release only includes small, backwards compatible
   changes.
 
-
 Currently Supported Versions
 ----------------------------
 
-If you are using an unsupported version of ``mongosync``, you may be
-asked to upgrade in order to receive support.
+MongoDB only supports the latest patch release of each major
+version.  For example, ``mongosync`` {+latest-version+} is the
+latest patch release of {+version+}. If you are using an
+unsupported version of ``mongosync``, you may be asked to
```

Comment:
[nit]

This ought to be "might" on line 36 according to the style guide: https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#std-label-helping-verbs

Reply:
Fixed